### ISSUE TITLE
Macro Composition

### ISSUE THREAD

**Issue by h3rald** [h3rald/glyph#179]
```
  xml[table[...]]
  --> xml/table[...]
  ?[not[eq[something[...]|something_else[...]]]]
  --> ?/not/eq[something[...]|something_else[...]]
```

It should be implemented at parser level.

**Comment by h3rald** [comment]
Implemented macro composition
- Closed by fd399179ceb10b149988af8c2c5ac5c4809b56ea.

**Comment by h3rald** [comment]
Change: =tag[...] no longer supported, use xml/tag[...]
